packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -222,31 +245,28 @@ class AttestationRequestHandler {
         this.identifier,
         transaction
       )
-      const provider = smsProviderFor(this.attestationRequest.phoneNumber)
-
-      if (!provider) {
-        await attestationRecord.update(
-          { status: AttestationStatus.UNABLE_TO_SERVE, smsProvider: SmsProviderType.UNKNOWN },
-          { transaction, logging: this.sequelizeLogger }
-        )
-        await transaction.commit()
-        Counters.attestationRequestsUnableToServe.inc()
-        return attestationRecord
-      }
 
       try {
-        await provider.sendSms(this.attestationRequest.phoneNumber, textMessage)
-        this.logger.info('Sent sms')
+        const sentVia = await startSendSms(
+          this.attestationRequest.phoneNumber,
+          textMessage,
+          () => {
+            void this.persistFinalAttesationResult(AttestationStatus.FAILED)
+          },
+          () => {
+            void this.persistFinalAttesationResult(AttestationStatus.COMPLETE)
```

Comment:
What is the different of AttestationStatus.COMPLETE/SENT? I don't think this service has any awareness of completion onchain now, but I'd assume that's what the implication for that state is.

Should this be SENT?

Reply:
`COMPLETE` was a pre-existing value, but I'm using it to indicate we got a "success" delivery notification